scrapper.py

```diff
@@ -48,14 +74,33 @@ def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(base_path)
```

Comment:
will work for empty dictionaries only. read the docs: https://docs.python.org/3/library/os.html#os.rmdir